src/lib/constants.ts

```diff
@@ -44,7 +44,14 @@ export const PAYMENT_TOKEN_FAUCET_URL = "https://faucet.paradigm.xyz";
 export const BETA_AGREEMENT_KEY = "storedBetaAgreement";
 export const STORAGE_WORKER_ENDPOINT =
   "https://storage-workers.geo-web.workers.dev";
-export const IPFS_GATEWAY = import.meta.env.VITE_IPFS_GATEWAY;
+export const IPFS_GATEWAYS = [
```

Comment:
How does it use multiple gateways? Does it select on of these by random? Or fallback?

Reply:
They are used in order, when one fails it fallback to the next in the array.
The library by default uses https://trustless-gateway.link as the first one and here is left in the first place, but is blocked by my ISP.
I added Pinata as second as it seems to be the most reliable relative to the others.